src/components/Header/Header.js

```diff
@@ -33,9 +37,10 @@ const Header = ({
   isAuthenticated,
   onClick,
   onLogout,
-  user = {
-    name: "Display Name",
-    image: { url: "" }
+  userProfile: {
+    imageUrl = "/images/user.png",
+    displayName = "Display Name",
```

Comment:
My initial thinking here is to default to displaying their email address (which we know we have) until they set a display name in the profile.  Likely a much better default than the static 'Display Name'.  Maybe @stubelshe has some input on this as well.

Reply:
Originally it was going to be their "@account" name until they had a display name entered.

Reply:
They don't necessarily have an eos account connected, however... Should we degrade from displayname to account to email?

Reply:
Right, we have changed the account creation process since we first designed this. Seems a bit odd now that things have changed to even put the wallet display name up there. Maybe not, we'll just have to see. I say we populate with the email address by default until a user updates their display name.